src/components/Test/Quote.test.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+import Quote from '../Quote';
+
+describe('Quote component', () => {
+  it('should render quote and author correctly', () => {
+    const quoteText = 'This is a test quote';
+    const authorName = 'Test Author';
+    const { getByText, getByAltText } = render(
+      <Quote quote={quoteText} author={authorName} />,
+    );
```

Comment:
- [ ] Please make sure to write tests for the `Calculator` and `Header` components as requested in the project requirements to write tests for all the components.

![image](https://github.com/AsumVictor/Math-magician/assets/83803414/8a0e8625-3ff8-442c-a1bb-7f0659e8ffde)

Reply:
We have resolved this. Thank you very much!